<launch>
  <node name="rover" pkg="arm_mimic_capstone" type="rover">
    <param name="leap_output_pose_topic" value="/leap_hand_pose"/>
    <param name="output_publish_topic" value="/turtlebot/stamped_cmd_vel_mux/input/joystick_cmd_vel">
  </node>


</launch>
